spec/features/medicaid_application_with_maximum_info_spec.rb

```diff
@@ -104,11 +104,11 @@
     on_pages "Income & Expense Amounts" do
       click_on "Next"
 
-      fill_in "Your Current Job (#1)", with: 100
-      fill_in "Your Current Job (#2)", with: 50
-      fill_in "Your Current Job (#3)", with: 25
-      fill_in "Your Self-Employment", with: 100
-      fill_in "Your Unemployment", with: 100
+      fill_in "Job 1", with: 100
```

Comment:
these will change with the new page styles that are on `master` (we are using `id`s now since all income inputs technically have the same label and are only differentiated by a header element)